lib/foreman_ansible/engine.rb

```diff
@@ -76,13 +76,8 @@ class Engine < ::Rails::Engine
       if Rails.env.test? ||
          foreman_version.major.to_i == 1 && foreman_version.minor.to_i < 13
         ::Foreman::Plugin.fact_importer_registry.register(:ansible, ForemanAnsible::FactImporter)
```

Comment:
I'm pretty sure we can drop this as well, foreman_ansible requires a much newer foreman already